src/tools/agent-ctl/README.md

```diff
@@ -263,3 +263,121 @@ running.
+  ```
+
+#### Check the agent is running with QEMU
+
+```bash
+$ cargo run -- -l trace connect --no-auto-values --bundle-dir "${bundle_dir}" --server-address "sock://${guest_cid}:${agent_vsock_port}" -c Check
+```
+
+#### Show guest environment details with QEMU
+
+```bash
+$ cargo run -- -l trace connect --no-auto-values --bundle-dir "${bundle_dir}" --server-address "sock://${guest_cid}:${agent_vsock_port}" -c GetGuestDetails
+```
+
+#### Show sandbox metrics with QEMU
+
+```bash
+$ cargo run -- -l trace connect --no-auto-values --bundle-dir "${bundle_dir}" --server-address "vsock://${guest_cid}:${agent_vsock_port}" -c GetMetrics
+```
+
+#### Show container statistics with QEMU
+
+```bash
+$ cargo run -- -l trace connect --no-auto-values --bundle-dir "${bundle_dir}" --server-address "vsock://${guest_cid}:${agent_vsock_port}" -c "StatsContainer json://{\"container_id\": \"${container_id}\"}"
+```
+
+#### Pause a running container with QEMU
+
+```bash
+$ cargo run -- -l trace connect --no-auto-values --bundle-dir "${bundle_dir}" --server-address "sock://${guest_cid}:${agent_vsock_port}" -c "PauseContainer json://{\"container_id\": \"$container_id\"}"
+```
+
+#### Resume a paused container with QEMU
+
+```bash
+$ cargo run -- -l trace connect --no-auto-values --bundle-dir "${bundle_dir}" --server-address "sock://${guest_cid}:${agent_vsock_port}" -c "ResumeContainer json://{\"container_id\": \"$container_id\"}"
+```
+
+#### Destroy a running container with QEMU
+
+```bash
+$ cargo run -- -l trace connect --no-auto-values --bundle-dir "${bundle_dir}" --server-address "sock://${guest_cid}:${agent_vsock_port}" -c "RemoveContainer json://{\"container_id\": \"$container_id\"}"
+```
+
+> **Note:**
+>
+> Only run this on a test or development system!
+
+#### Destroy a running sandbox with QEMU
+
+```bash
+$ cargo run -- -l trace connect --no-auto-values --bundle-dir "${bundle_dir}" --server-address "vsock://${guest_cid}:${agent_vsock_port}" -c 'DestroySandbox'
+```
+
+> **Notes:**
+>
+> - Only run this on a test or development system!
+> - This will destroy the sandbox and all resources associated with it
+>   (including all containers and the VM that hosts the agent).
+> - You cannot create a sandbox this way since the Kata Containers
+>   runtime and agent will already have created one inside the QEMU
+>   VM.
+>
+>   See [Create a sandbox manually](#create-a-sandbox-manually).
+
+### Local examples
+
+> **Note:**
+>
+> These examples assume you have
+> [already started the agent manually](#use-a-unix-abstract-domain-socket).
+
+#### Create a sandbox manually
+
+```bash
+$ sandbox_id="foo"
+
+# Critical to clear up from any previous run
+$ sudo umount /run/sandbox-ns/*
+
+$ server_addr='unix://@/tmp/foo.socket'
+
+$ cargo run -- -l trace connect --no-auto-values --bundle-dir "${bundle_dir}" --server-address "${server_addr}" -c "CreateSandbox json://{\"sandbox_id\": \"$sandbox_id\"}"
```

Comment:
Wasn't able to successfully run these last two commands:
```
ERROR: Failed to connect to Unix Domain abstract socket
```

Reply:
That suggests you didn't start the agent first. I've just re-tested this and if you install the agent, this will work:

```bash
$ sudo KATA_AGENT_SERVER_ADDR=unix:///tmp/foo.socket KATA_AGENT_LOG_LEVEL=trace kata-agent
```